src/components/HomePage.js

```diff
@@ -1,102 +1,112 @@
-import React, { Component } from 'react'
-import {Container, Row, Col} from 'reactstrap'
-import {Link} from 'react-router-dom'
-import {data} from '../helpers/services'
-import destinations from '../assets/destinations.png'
-import previous from '../assets/previous.png'
-import next from '../assets/next.png'
-import Card from './Card'
-import Navigation from './Navigation'
-import Footer from './Footer'
+import React, { Component } from "react";
+import { Container, Row, Col } from "reactstrap";
+import { Link } from "react-router-dom";
+import { data } from "../helpers/services";
+import destinations from "../assets/destinations.png";
+import previous from "../assets/previous.png";
+import next from "../assets/next.png";
+import Card from "./Card";
+import Navigation from "./Navigation";
+import Footer from "./Footer";
 
 class NewHomePage extends Component {
```

Comment:
be a bit consistent and choose between class components and functional components.